### ISSUE TITLE
Retrying feature

### ISSUE THREAD

**Issue by naufalad** [gtalarico/pyairtable#206]
Hi, 

I just upgraded pyairtable via pip to 1.3.0, but it seems the retrying function is not working as expected.
I can't even call `from pyairtable import retry_strategy`
And when I opened the library, can't find any code related to retry_strategy

Any solution? As it says it should be available on version 1.3.0
![image](https://user-images.githubusercontent.com/33115829/200237049-4f7b297d-1df8-48bf-8271-7db6a40e28a0.png)

**Comment by gtalarico** [comment]
I think it was a mistake in docs.
Can you try upgrade to latest?
https://github.com/gtalarico/pyairtable/issues/201#issue-1421993182

**Comment by naufalad** [comment]
@gtalarico yeah I could do that, but hope this retrying feature could be tagged as a version in the near future

**Comment by gtalarico** [comment]
should be in latest tagged

**Comment by gtalarico** [comment]
Should be fixed to 1.4
<img width="643" alt="image" src="https://user-images.githubusercontent.com/9513968/215442496-338d2ecd-3d0f-455a-9e52-e7b9cfcefcda.png">
